src/core/sprites/webgl/SpriteRenderer.js

```diff
@@ -226,7 +227,8 @@ export default class SpriteRenderer extends ObjectRenderer
         let currentGroup = groups[0];
         let vertexData;
         let uvs;
-        let blendMode = sprites[0].blendMode;
+        let blendMode = premultiplyBlendMode[
+            Number(sprites[0]._texture.baseTexture.premultipliedAlpha)][sprites[0].blendMode];
```

Comment:
Rather than `Number(x)` I would do a ternary since `Number(undefined)` is `NaN`.

```js
let blendModeIndex = sprites[0]._texture.baseTexture.premultipliedAlpha ? 1 : 0;
let blendMode = premultiplyBlendMode[blendModeIndex][sprites[0].blendMode];
```